packages/block-library/src/cover/index.php

```diff
@@ -40,7 +40,7 @@ function render_block_core_cover( $attributes, $content ) {
 		$object_position = '';
 		if ( isset( $attributes['focalPoint'] ) ) {
 			$object_position = round( $attributes['focalPoint']['x'] * 100 ) . '%' . ' ' .
-			round( $attributes['focalPoint']['x'] * 100 ) . '%';
+			round( $attributes['focalPoint']['y'] * 100 ) . '%';
```

Comment:
I’m might be missing something, but just rounding value here shouldn’t fix the issue. I think we should also combine it into $object_position variable.

Reply:
Oh it’s multine contact. Never mind 🙇

Reply:
So previously, `$object_position = (rounded x value) (rounded x value)`. Now it is `$object_position = (rounded x value) (rounded y value)`.